### ISSUE TITLE
Failed to verify code signature 

### ISSUE THREAD

**Issue by QusaySalamin** [rileytestut/AltStore#1070]
I have an issue on altstore whenever i try to refresh my apps or add new ones it doesn’t work and only altstore refreshes and it gives me this error
![E74DE6B4-B9F6-4FA4-839A-75B08E7A0C3C](https://user-images.githubusercontent.com/115823515/195910425-d00e3e1c-019a-48ea-9b1b-ca5c9b1be841.png)

**Comment by lonkelle** [comment]
What app are you trying to install?

**Comment by shanegillio** [comment]
would you say this is the same issue as some of the other framework issues you've seen @lonkelle ? wondering if we can just give some attention to that and fix things up

**Comment by lonkelle** [comment]
@shanegillio Correct, this is the majority if not all of them (would have to check). The CydiaSubstrate framework WORKS in sideloaded apps but unfortunately it requires a lot of work to support the framework that only one person has accomplished it in code (the Scarlet project, which is closed source). 

If anyone could match that, I bet it'd be Riley, but he's at a disadvantage a bit given his dislike of jailbreak coding. Which this wouldn't be jailbreak coding, but it would be dealing with the substrate, making it compatibility in jailed mode. It's a tough undertaking but does consist of a lot of IPAs.

**Comment by shanegillio** [comment]
ok maybe we look into this as a larger implementation then, not necessarily something we will have a quick fix for out right away

**Comment by FieryFlames** [comment]
Hi, I'm running into the same issue (or at least similar, 0xe8008015) when trying to sideload a IPA injected with [vendetta-mod/VendettaTweak](https://github.com/vendetta-mod/VendettaTweak) and [theos/orion](https://github.com/theos/orion). Sideloadly can sideload the IPA fine.
